nautobot/docs/additional-features/job-hooks.md

```diff
@@ -0,0 +1,52 @@
+# Job hooks
+
+A job hook is a mechanism for automatically starting a [job](./jobs.md) when an object is changed. Job hooks are similar to [webhooks](../models/extras/webhook.md) except change events initiate a `JobHookReceiver` job instead of a web request. Job hooks are configured in the web UI under Jobs > Job Hooks.
```

Comment:
```suggestion
A Job Hook is a mechanism for automatically starting a [job](./jobs.md) when an object is changed. Job Hooks are similar to [webhooks](../models/extras/webhook.md) except that an object change event initiates a `JobHookReceiver` job instead of a web request. Job hooks are configured in the web UI under **Jobs > Job Hooks**.
```